commands/local_proxy_get_url.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021-present Fabien Potencier <fabien@symfony.com>
+ *
+ * This file is part of Symfony CLI project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/symfony-cli/console"
+	"github.com/symfony-cli/symfony-cli/local/pid"
+	"github.com/symfony-cli/terminal"
+)
+
+var localProxyGetUrlCmd = &console.Command{
+	Category: "local",
+	Name:     "proxy:get-url",
+	Aliases:  []*console.Alias{{Name: "proxy:get-url"}},
+	Usage:    "Get the local proxy server URL",
+	Flags: []console.Flag{
+		dirFlag,
+	},
+	Action: func(c *console.Context) error {
+		pidFile := pid.New("__proxy__", nil)
+		if !pidFile.IsRunning() {
+			terminal.Println("    <error>Not Running</>")
+			os.Exit(1)
```

Comment:
you can directly return an error here, `symfony-cli/console` will take care of displaying the error with a nice red error box (à la Symfony) on stderr and have an exit code of 1